.github/workflows/package.yml

```diff
@@ -28,11 +28,21 @@ jobs:
               pyinstaller tabcmd-windows.spec --clean --noconfirm --distpath ./dist/windows
             OUT_FILE_NAME: tabcmd.exe
             ASSET_MIME: application/vnd.microsoft.portable-executable
+          - os: macos-13
+            TARGET: macos
+            CMD_BUILD:  >
+              pyinstaller tabcmd-mac.spec --clean --noconfirm --distpath ./dist/macos/
+            BUNDLE_NAME: tabcmd.app
+            OUT_FILE_NAME: tabcmd.app
+            APP_BINARY_FILE_NAME: tabcmd
+            ASSET_MIME: application/zip
           - os: macos-latest
             TARGET: macos
             CMD_BUILD:  >
-              pyinstaller tabcmd-mac.spec --clean --noconfirm --distpath ./dist/macos
+              pyinstaller tabcmd-mac.spec --clean --noconfirm --distpath ./dist/macos/
+            BUNDLE_NAME: tabcmd_arm64.app
             OUT_FILE_NAME: tabcmd.app
```

Comment:
should this be tabcmd_arm64.app?

Reply:
I kept the App name as tabcmd.app same across Mac platform as tabcmd.app, just differentiating it in github release artifacts for easy identification while looking to download